Ginger/Ginger/UserControlsLib/UCListView/UcListView.xaml

```diff
@@ -51,7 +51,7 @@
         </Border>
 
         <Ginger:ucTagsViewer x:Name="xTagsFilter" Grid.Row="2" HorizontalAlignment="Left" VerticalAlignment="Center" AddTags="False"/>
-        <Label Margin="5,0,0,0" x:Name="SelectionTitle" Height="40" Grid.Row="2" HorizontalAlignment="Left" VerticalAlignment="Center" Style="{StaticResource $SubHeaderLabelStyle}" FontSize="18" Visibility="Hidden"/>
+        <Label Margin="5,0,0,0" x:Name="SelectionTitle" Height="40" Grid.Row="2" HorizontalAlignment="Left" VerticalAlignment="Center" Style="{StaticResource $SubHeaderLabelStyle}" FontWeight="Normal"  FontSize="18" Visibility="Hidden"/>
```

Comment:
This change will affect all placed wherever this label is being used. can you make the change specific to the page where you need it?

Reply:
ok

Reply:
Implemented the required changes